dockerfiles/hwsc-user-svc/build_push_psql_image.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# combine all *.up.sql files into one sql file for docker build to consume
+cat ../../db-migrations/hwsc-user-svc/test/psql/*.up.sql > user_svc_schema.sql
+
+# build an image from a docker file, -t = name and optionally a tag in the 'name:tag' format
+# docker build [OPTIONS] PATH
+docker build --build-arg db_name=$hosts_postgres_db -t hwsc/hwsc-user-svc-psql:int .
```

Comment:
This should be tagged `:test-int`.

Reply:
`:int` tag will be used for deployed dev environment

Reply:
Pushed changes to origin and deleted :int tag from docker hub